CP05MOAS-GL559/CP05MOAS-GL559_D00004_ingest.csv

```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/D00004/merged-from-glider/cp_*.mrg,CP05MOAS-GL559-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/D00004/merged-from-glider/cp_*.mrg,CP05MOAS-GL559-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/D00004/merged-from-glider/cp_*.mrg,CP05MOAS-GL559-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/D00004/merged-from-glider/cp_*.mrg,CP05MOAS-GL559-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/D00004/merged-from-glider/cp_*.mrg,CP05MOAS-GL559-05-PARADM000,telemetered,Available,
```

Comment:
looks good; just wondered if you need to add a note that the DOSTA was top-mount orientation?